osu.Game/Scoring/ScoreManager.cs

```diff
@@ -71,8 +72,9 @@ protected override ScoreInfo CreateModel(ArchiveReader archive)
             }
         }
 
-        protected override IEnumerable<string> GetStableImportPaths(Storage stableStorage)
-            => stableStorage.GetFiles(ImportFromStablePath).Where(p => HandledExtensions.Any(ext => Path.GetExtension(p)?.Equals(ext, StringComparison.OrdinalIgnoreCase) ?? false));
+        protected override IEnumerable<string> GetStableImportPaths(StableStorage stableStorage)
+            => stableStorage.GetFiles(ImportFromStablePath).Where(p => HandledExtensions.Any(ext => Path.GetExtension(p)?.Equals(ext, StringComparison.OrdinalIgnoreCase) ?? false))
+                            .Select(path => stableStorage.GetFullPath(path));
```

Comment:
is moving these `GetFullPath()` calls out from the base `ArchiveModelManager` necessary? would rather have them in one place than all over.

Reply:
Yeah moving these `GetFullPath()` calls out of the base class is necessary because the storage returned by `GetSongStorage()` may not have a similar base path as the base path of the StableStorage (ie: when the songs folder is located on another disk) and as such calling  `GetFullPath()` with the `StableStorage` on paths from the song directory may completely scramble them in that case.

Reply:
`GetStableImportPaths()` should have xmldoc that all paths returned must be absolute in that case.